arch/arm64/core/smp.c

```diff
@@ -180,7 +181,7 @@ void arch_secondary_cpu_init(int cpu_num)
 
 #ifdef CONFIG_SMP
 
-static void broadcast_ipi(unsigned int ipi)
+static void send_ipi(unsigned int ipi, uint32_t cpu_bitmap)
```

Comment:
Style nit: the platform maintainers might prefer to see each arch get support added in a separate patch instead of all at once, but that's not really for me to say.